### PR TITLE
Feature/cv Àlex Barba

### PR DESCRIPTION
**Pull request by lexbarba**

### CODE REVIEW

File: staff/alex-barba/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Àlex Barba - CV</title>
+</head>
+
+<body>
+    <header>
+        <h1>Àlex Barba <img src="myphoto.jpg" alt="àlex barba photo" height="60" width="42"></h1> 
+        <p>C/Mercat 2 Esc B 3 3, 08750 - Molins de Rei, BCN</p>
```
Comment (boversauros) [inline staff/alex-barba/cv/index.html]: it will be nice to use the tag `address` for the adress information

Comment (boversauros) [reply staff/alex-barba/cv/index.html]: https://www.w3schools.com/tags/tag_address.asp

Comment (lexbarba) [reply staff/alex-barba/cv/index.html]: Thx! Didn't know about it. Highly appreciated!

File: staff/alex-barba/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        <h1>Àlex Barba <img src="myphoto.jpg" alt="àlex barba photo" height="60" width="42"></h1> 
+        <address>C/Mercat 2 Esc B 3 3, 08750 - Molins de Rei, BCN</address>
+        <p>Mobile: <a href="tel:0034618610187">+34 618 610 187</a>, email: <a href="mailto:alexander.barba@gmail.com">alexander.barba@gmail.com</a>, LinkedIn: àlex-barba </p>
+    </header>
+    <nav>
+            <a href="#we">Work Experience</a> |
+            <a href="#ed">Education</a> |
+            <a href="#la">Language</a> |
+            <a href="#sk">Skills</a>
+          </nav>
```
Comment (danieltolmon) [inline staff/alex-barba/cv/index.html]: Cool adding nav

File: staff/alex-barba/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        <address>C/Mercat 2 Esc B 3 3, 08750 - Molins de Rei, BCN</address>
+        <p>Mobile: <a href="tel:0034618610187">+34 618 610 187</a>, email: <a href="mailto:alexander.barba@gmail.com">alexander.barba@gmail.com</a>, LinkedIn: àlex-barba </p>
+    </header>
+    <nav>
+            <a href="#we">Work Experience</a> |
+            <a href="#ed">Education</a> |
+            <a href="#la">Language</a> |
+            <a href="#sk">Skills</a>
+    </nav>
+    <section>
```
Comment (johanaludwig) [inline staff/alex-barba/cv/index.html]: Nice work Àlex! maybe you can add a <main> tag.
Here more information if you need: https://developer.mozilla.org/es/docs/Web/HTML/Elemento/main

Comment (lexbarba) [reply staff/alex-barba/cv/index.html]: You are right. Thnx johana!

**Pull request merged**